### ISSUE TITLE
Cannot read property 'setAutoLogAppEventsEnabled' of undefined

### ISSUE THREAD

**Issue by juanchofelipe** [NativeScript/nativescript-facebook#81]
### Which platform(s) does your issue occur on?
- iAndroid, emulator and device

### Please, provide the following version numbers that your issue occurs with:

- CLI: 3.4.0
- Cross-platform modules: 3.3.0
- Runtime(s): tns-android: 3.4.0
- Plugin(s): 

"dependencies": {
		"@angular/animations": "~4.4.1",
		"@angular/common": "~4.4.1",
		"@angular/compiler": "~4.4.1",
		"@angular/core": "~4.4.1",
		"@angular/forms": "~4.4.1",
		"@angular/http": "~4.4.1",
		"@angular/platform-browser": "~4.4.1",
		"@angular/router": "~4.4.1",
		"nativescript-angular": "~4.4.0",
		"nativescript-facebook": "^2.1.0",
		"nativescript-image-swipe": "^2.1.0",
		"nativescript-ng2-carousel": "0.0.5",
		"nativescript-pro-ui": "^3.2.0",
		"nativescript-theme-core": "~1.0.2",
		"reflect-metadata": "~0.1.8",
		"rxjs": "~5.4.2",
		"tns-core-modules": "~3.3.0",
		"zone.js": "~0.8.2"
	},
	"devDependencies": {
		"babel-traverse": "6.26.0",
		"babel-types": "6.26.0",
		"babylon": "6.18.0",
		"lazy": "1.0.11",
		"nativescript-dev-typescript": "~0.5.0",
		"typescript": "~2.4.2"
	}

### Please, tell us how to recreate the issue in as much detail as possible. 
I added the plugin and followed the instructions for angular and got that error message

System.err: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.nativescript.ChupitosUser/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: 
System.err: Calling js method onCreate failed
System.err: 
System.err: TypeError: Cannot read property 'setAutoLogAppEventsEnabled' of undefined
System.err: File: "file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/nativescript-facebook/login-manager.js, line: 14, column: 28
System.err: 
System.err: StackTrace: 
System.err: 	Frame: function:'init', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/nativescript-facebook/login-manager.js', line: 14, column: 29
System.err: 	Frame: function:'', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/app.module.js', line: 26, column: 16
System.err: 	Frame: function:'Observable.notify', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/data/observable/observable.js', line: 103, column: 23
System.err: 	Frame: function:'ActivityCallbacksImplementation.notifyLaunch', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 621, column: 36
System.err: 	Frame: function:'ActivityCallbacksImplementation.onCreate', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 585, column: 29
System.err: 	Frame: function:'NativeScriptActivity.onCreate', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/ui/frame/activity.js', line: 20, column: 25
System.err: 
System.err: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2684)
System.err: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2751)
System.err: 	at android.app.ActivityThread.-wrap12(ActivityThread.java)
System.err: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1496)
System.err: 	at android.os.Handler.dispatchMessage(Handler.java:102)
System.err: 	at android.os.Looper.loop(Looper.java:154)
System.err: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
System.err: 	at java.lang.reflect.Method.invoke(Native Method)
System.err: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
System.err: Caused by: com.tns.NativeScriptException: 
System.err: Calling js method onCreate failed
System.err: 
System.err: TypeError: Cannot read property 'setAutoLogAppEventsEnabled' of undefined
System.err: File: "file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/nativescript-facebook/login-manager.js, line: 14, column: 28
System.err: 
System.err: StackTrace: 
System.err: 	Frame: function:'init', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/nativescript-facebook/login-manager.js', line: 14, column: 29
System.err: 	Frame: function:'', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/app.module.js', line: 26, column: 16
System.err: 	Frame: function:'Observable.notify', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/data/observable/observable.js', line: 103, column: 23
System.err: 	Frame: function:'ActivityCallbacksImplementation.notifyLaunch', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 621, column: 36
System.err: 	Frame: function:'ActivityCallbacksImplementation.onCreate', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 585, column: 29
System.err: 	Frame: function:'NativeScriptActivity.onCreate', file:'file:///data/data/org.nativescript.ChupitosUser/files/app/tns_modules/tns-core-modules/ui/frame/activity.js', line: 20, column: 25
System.err: 
System.err: 	at com.tns.Runtime.callJSMethodNative(Native Method)
System.err: 	at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1088)
System.err: 	at com.tns.Runtime.callJSMethodImpl(Runtime.java:970)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:957)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:941)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:933)
System.err: 	at com.tns.NativeScriptActivity.onCreate(NativeScriptActivity.java:13)
System.err: 	at android.app.Activity.performCreate(Activity.java:6684)
System.err: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
System.err: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2637)
System.err: 	... 9 more
ActivityManager: Process org.nativescript.ChupitosUser (pid 1791) has died
ActivityManager: cleanUpApplicationRecord -- 1791
^CExecuting after-watch hook from /home/juanfe/Projects/ChupitosUser/hooks/after-watch/nativescript-dev-typescript.js
Stopping tsc watch
Executing after-watch hook from /home/juanfe/Projects/ChupitosUser/hooks/after-watch/nativescript-dev-typescript.js

**Comment by juanchofelipe** [comment]
Hi! I just could resolve the issue, I deleted the next 3 folders: /platforms /hooks /node_modules folders

And then I executed:

`npm cache clean --force`

After that, I run the project and it worked

**Comment by VladimirAmiorkov** [comment]
Hi @juanchofelipe ,

Good to hear, if this issue is resolved I will close it for now. If you find yourself again at this issue please feel free to reopen it again.

**Comment by arnaudpoullet** [comment]
Hi,
I just got the same error with Nativescript-Vue (run as android preview) following the vue demo-app.

I tried removing the 3 folders and clearing my npm cache but it didn't help.

The undefined variable is "com.facebook.FacebookSdk" and it's only mentionned 3 times in the plugin in the "login-manager.android.js" file.

Is it possible I have to manually load some android facebooksdk dependency ?
I'm using the latest version of everything.
